ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -294,100 +213,36 @@ if [[ "$CNF_BRANCH" == *"4.13"* ]] || [[ "$CNF_BRANCH" == *"master"* ]]; then
 fi
 cp "$SKIP_TESTS_FILE" "${ARTIFACT_DIR}/"
 
-# run cnf-tests by feature in a thread on a free worker node
-for feature in ${FEATURES}; do
-    log_file="${ARTIFACT_DIR}/deploy_and_test_${feature}.log"
-    rm -f "${log_file}"
-
-    feature_nodes=""
-    num_of_free_nodes=0
-    num_of_required_nodes=1
-    if [[ "${feature}" == "ovs_qos" ]]; then
-        num_of_required_nodes=2
-    fi
-    while [ ${num_of_free_nodes} -lt ${num_of_required_nodes} ]; do
-        for node in ${test_nodes}; do
-            node_ready_file="${node}_ready.txt"
-            if [ -f "${node_ready_file}" ]; then
-                rm -f "${node_ready_file}"
-                feature_nodes="${feature_nodes} ${node}"
-                num_of_free_nodes=$((num_of_free_nodes+1))
-            fi
-            if [ ${num_of_free_nodes} -eq ${num_of_required_nodes} ]; then
-                (deploy_and_test "${feature}" "${feature_nodes}" || true) 2>&1 | tee "${log_file}" &
-                break
-            fi
-        done
-        sleep 10
-    done
-done
-wait
-
-# cleanup
-for node in ${test_nodes}; do
-    rm -f "${node}_ready.txt"
-done
-rm -rf node/
+export ROLE_WORKER_CNF="worker-cnf"
```

Comment:
that's the default label so no need to set it

Reply:
ack